### PR TITLE
add docs for superglooo istio install

### PR DESCRIPTION
**Pull request by ilackarms**
depends on a new release of supergloo, which is blocked by a bugfix i'm working on.
do not merge yet, but feel free to review!

### CODE REVIEW

File: supergloo/docs/mesh/install.md [hunk 1]
```diff
@@ -0,0 +1,264 @@
...
+istio-cleanup-secrets-xd6f8               0/1       Completed           0          27s
+istio-galley-6c68c5dbcf-z6j5k             1/1       Running             0          27s
+istio-pilot-c5dddb4b9-nb6fd               2/2       Running             0          26s
+istio-policy-977d74ff4-669vk              2/2       Running             0          27s
+istio-sidecar-injector-6d8f88c98f-5f58t   1/1       Running             0          26s
+istio-telemetry-5f79796bf6-fl4sn          2/2       Running             0          27s
+istio-tracing-7596597bd7-lc92t            1/1       Running             0          26s
+prometheus-76db5fddd5-55r6d               1/1       Running             0          26s
+
+````
```
Comment (christian-posta) [inline supergloo/docs/mesh/install.md]: Think you got one too many single quotes here. At the moment, it renders like this:

image

File: supergloo/docs/mesh/install.md [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+First, enable autoamtic sidecar injection on the default namespace (or any namespace of your choosing):
+
+```bash
+kubectl label namespace default istio-injection=enabled
+```
+
+Next, create the bookinfo deployments and services:
+
+```bash
+kubectl apply -n default -f https://raw.githubusercontent.com/istio/istio/1.0.6/samples/bookinfo/platform/kube/bookinfo.yaml
```
Comment (christian-posta) [inline supergloo/docs/mesh/install.md]: need the `\` at the line separation

File: supergloo/docs/mesh/install.md [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+
+
+Next we'll deploy a `sleep` pod which we can use to execute commands inside
+the cluster. The first time we deploy, we'll run it outside the mesh to see
+what happens when we try to connect to mtls-enabled services.
+
+```bash
+# create a new namespace without injection enabled
+kubectl create ns not-injected
+kubectl apply -n not-injected -f https://raw.githubusercontent.com/istio/istio/1.0.6/samples/sleep/sleep.yaml
```
Comment (christian-posta) [inline supergloo/docs/mesh/install.md]: Should prob break up the line with `\`

File: supergloo/docs/mesh/install.md [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+weight: 2
+---
+
+# Overview
+
+SuperGloo can be used to install, upgrade, and uninstall a supported mesh.
+
+Currently supported meshes for installation:
+
+- Istio
```
Comment (marcogschmidt) [inline supergloo/docs/mesh/install.md]: A list with one entry looks a bit weird, maybe we can add other elements with "WIP" or "planned" labels. Even better, we could reference [the feature table in the main README](https://github.com/solo-io/supergloo#features-snapshot) (which we should update!).

**Comment by ilackarms** [comment]
tested working with supergloo `0.3.2`

**Pull request merged**